lib/core/src/Cardano/Wallet/Primitive/CoinSelection/Migration.hs

```diff
@@ -111,7 +111,7 @@ depleteUTxO feeOpts batchSize utxo =
       where
         noDust :: TxOut -> Maybe Coin
         noDust (TxOut _ c)
-            | c < dustThreshold feeOpts = Nothing
+            | c <= dustThreshold feeOpts = Nothing
```

Comment:
So this is the same issue in the Migration coin selection algorithm like in `coalesceDust` I believe. But I can't tell if this actually fixes #2207